core/server/public/members.js

```diff
@@ -8,8 +8,10 @@ Array.prototype.forEach.call(document.querySelectorAll('form[data-members-form]'
             errorEl.innerText = '';
         }
         form.classList.remove('success', 'invalid', 'error');
-        var input = event.target.querySelector('input[data-members-email]');
-        var email = input.value;
+        var email_input = event.target.querySelector('input[data-members-email]');
```

Comment:
We generally use camelCase rather than snake_case - would you be able to update? :relaxed: